### PR TITLE
doc/website: add circleci integrations

### PR DESCRIPTION
**Pull request by datdao**
https://github.com/ariga/atlas/assets/16095902/95ec3b96-b1f9-43e2-8521-e9f402fedab5

### CODE REVIEW

File: doc/md/integrations/circleci-orbs.mdx [hunk 1]
```diff
@@ -0,0 +1,544 @@
...
+* `dev_url` - The URL of the dev-database to use for analysis.  For example: `mysql://root:pass@localhost:3306/dev`.
+  Read more about [dev-databases](https://atlasgo.io/concepts/dev-database).
+* `config` - The path to the Atlas configuration file.  By default, Atlas will look for a file
+  named `atlas.hcl` in the current directory. For example, `file://config/atlas.hcl`.
+  Learn more about [Atlas configuration files](https://atlasgo.io/atlas-schema/projects).
+* `env` - The environment to use from the Atlas configuration file.  For example, `dev`.
+* `github_token_env` (required) - The GitHub token to use for authentication. Defaults to `GITHUB_TOKEN` environment variable.
+* `github_repository_env` (required) - The GitHub repository to use for authentication. Defaults to `GITHUB_REPOSITORY` environment variable.
+
+:::warning
```
Comment (rotemtam) [inline doc/md/integrations/circleci-orbs.mdx:542]: This turns out as "DANGER" , which is a bit off-putting. Maybe info?

Comment (rotemtam) [reply doc/md/integrations/circleci-orbs.mdx:542]: And please explain what they are and what they are used for

Comment (datdao) [reply doc/md/integrations/circleci-orbs.mdx]: I changed the info and also edited the description of `github_repository_env` to more clear
```
* `github_token_env` (required) - The GitHub token to use for authentication.
  Defaults to `GITHUB_TOKEN` environment variable.
* `github_repository_env` (required) - The GitHub repository name that will be used for reporting lint results.
  Defaults to `GITHUB_REPOSITORY` environment variable.
```

**Comment by datdao** [comment]
Hi @rotemtam, any updates on this PR?

**Review by rotemtam — APPROVED** [review]

**Pull request merged**